src/components/accounts/SetRecoveryInfoForm.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+import { Form, Input, Button } from 'semantic-ui-react'
+import PhoneInput from 'react-phone-number-input'
+
+import ProblemsImage from '../../images/icon-problems.svg'
+import CheckBlueImage from '../../images/icon-check-blue.svg'
+
+import styled from 'styled-components'
+
+const RecoveryInfoForm = styled(Form)`
+   &&& input {
```

Comment:
is it normal in react to have css inline? I definitely prefer separate css files

Reply:
> is it normal in react to have css inline? I definitely prefer separate css files

This is somewhat common in react for component-level styles that aren't going to be reused elsewhere to avoid css anti-pattern of overriding, then overriding again with `!important`. I'm not personally a fan, but this is pretty widely accepted.

Reply:
That being said, this is a big enough block of style that I would argue it should be taken out of here.

Reply:
that's normal pattern, the way to refactor it is to cut components into smaller pieces

the good part about this is that it's scoped by component, so very easy to manage vs global styles which can get applied to whatever